packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -355,6 +359,64 @@ const backupScreens = (Navigator: typeof Stack) => (
   </>
 )
 
+const fiatExchangesAmountScreenOptions = ({
```

Comment:
Given how giant this file is becoming, I want to re-suggest moving these out of here

Reply:
Yeah I think putting this in the screen component file, or a separate file entirely is a good idea